server.py

```diff
@@ -97,23 +111,56 @@ def push_update_success_as_discord_embed(
         repo_name = prefix + " " + repo_name
         # do a gray color if we are sending "not real" embeds
         color = 0x99AAB5
-
+    commit = result.commit
+    branch = commit.branch or getattr(repo_config, 'branch', 'main')
+    commit_id = commit.id[:7] if commit.id else 'unknown'
+    commit_url = commit.url or 'https://github.com/SCE-Development/'
+    commit_message = commit.message or 'No commit message'
```

Comment:
if we want this to be the default, move 'no commit message' to the dataclass definition so we dont need to use `or` here